packages/tools/devtools/devtools-core/src/test/DataVisualizerGraph.test.ts

```diff
@@ -194,17 +194,20 @@ describe("DataVisualizerGraph unit tests", () => {
 		expect(childCounterTree).to.deep.equal(expectedChildCounterTree);
 
 		const childCellTree = await visualizer.render(sharedCell.id);
-		const expectedChildCellTree: FluidObjectTreeNode = {
+		const expectedChildCellTree: FluidObjectValueNode = {
 			fluidObjectId: sharedCell.id,
-			children: {
-				data: {
-					value: "Hello world",
-					typeMetadata: "string",
-					nodeKind: VisualNodeKind.ValueNode,
-				},
-			},
+			value: "Hello world",
 			typeMetadata: "SharedCell",
-			nodeKind: VisualNodeKind.FluidTreeNode,
+			nodeKind: VisualNodeKind.FluidValueNode,
+			editProps: {
+				editTypes: [
```

Comment:
I believe the intention was for `editTypes: undefined` to indicate that any edit type is allowed. We should take advantage of that here so we don't have to edit this code when we add support for new edit types.

Reply:
My thinking was that:
1. Passing all of them makes it very easy to generate the dropdown since you just map each entry in editTypes to an option in the dropdown. 
2. Eventually we might add a distinction between "node" and "value" edits, but in the case that that doesnt happen, sharedCell won't be able to use "new row" or "new entry" edit types like sharedMatrix and SharedMap. 

What are your thoughts on this?

Reply:
> 1. Passing all of them makes it very easy to generate the dropdown since you just map each entry in editTypes to an option in the dropdown.

I would recommend just adding an "allOptions" (or something) constant on the view side, and use that when `editTypes` is `undefined`. That way we can define the "all" list in 1 place.

> 2. Eventually we might add a distinction between "node" and "value" edits, but in the case that that doesnt happen, sharedCell won't be able to use "new row" or "new entry" edit types like sharedMatrix and SharedMap.

I think we'll probably add those degrees of freedom via new properties under `editProps`, so they can be independent of the `editTypes` list.